packages/fxa-content-server/tests/functional/settings_v2/external_links.js

```diff
@@ -54,22 +54,24 @@ describe('external links', () => {
     );
   });
 
-  it('renders Subscriptions link in navigation when we are subscribed to a product', async () => {
+  it('renders Subscriptions link in navigation when we are subscribed to a product', function () {
```

Comment:
I think it can be `async`, just not an arrow function

Reply:
@dannycoates ah you're right good catch, updating!